static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -118,6 +118,7 @@ define([
             if (guardian.isEnhanced) {
                 ab.segmentUser();
                 ab.run();
+                ab.trackEvent();
```

Comment:
`run` means "run the tests" ! There's no `init` function in the A/B module at the moment, but maybe it'd be better to move these three calls into one, rather than having them inside the bootstrap module.